framework_crates/bones_framework/Cargo.toml

```diff
@@ -73,12 +73,20 @@ bones_schema = { version = "0.3", path = "../bones_schema", features = ["humanti
 bones_scripting = { version = "0.3", path = "../bones_scripting", optional = true }
 
 # Other
+anyhow              = "1.0"
+async-channel       = "1.7"
+bevy_tasks = { version = "0.11"}
+bytemuck               = "1.12"
+either                 = "1.8"
+futures-lite           = "1.12"
 glam      = "0.24"
-thiserror = "1.0"
-instant   = { version = "0.1", features = ["wasm-bindgen"] }
 hex       = "0.4"
-tracing   = "0.1"
+instant   = { version = "0.1", features = ["wasm-bindgen"] }
 noise     = "0.8"
+once_cell              = "1.17"
+rand                   = "0.8"
```

Comment:
```suggestion
```

We already have `turborand` in scope ( from the `bones_utils` crate ), so we should use that instead of rand. It looks like it's only used once for the matchmaker random seed.